recipes/xarray_mongodb/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xarray_mongodb
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xarray_mongodb/xarray_mongodb-{{ version }}.tar.gz
+  sha256: d55c766d9d320a6e83b1d49a0db4b0b0617972b6bca39078fd1da207d0a026dd
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.13
+    - pandas >=0.21
+    - dask >=1.1
+    - xarray >=0.10.4
+    - pymongo >=3.7
+
+test:
+  imports:
+    - xarray_mongodb
+  requires:
+    - pytest >=3.7
+    - pytest-asyncio >=0.10
+
+about:
+  home: https://github.com/AmphoraInc/xarray_mongodb
+  license: APACHE-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Store xarray objects in MongoDB
+
+extra:
+  recipe-maintainers:
+    - crusaderky
+    - gimperiale
```

Comment:
@gimperiale are you willing to be a maintainer?

Reply:
We're the same person. This is the corporate account whereas @crusaderky is the personal one.

Reply:
haha ok